src/ast-utils.ts

```diff
@@ -94,3 +92,62 @@ export function querySelectorAllTemplates(
   }
   return results;
 }
+
+/**
+ * The results of `queryAll` combined with `querySelectorAllTemplates`.
+ */
+export function querySelectorAllWithTemplates(
```

Comment:
Oh look at that!  Didn't know that was a thing, `dom5.childNodesIncludeTemplate`.  Done!